products/opensuse/templates.d/ltp

```diff
@@ -45,5 +45,194 @@
          {key => 'LTP_COMMAND_PATTERN', value => 'memcg(?!_stress)'},
          {key => 'LTP_TIMEOUT',         value => 1200},
          {key => 'START_AFTER_TEST',    value => 'install_ltp'}]
+       },
+       {
+        name     => 'setup_ltp_networking',
+        settings =>
+        [{key => 'DESKTOP',                 value => 'textmode'},
+         {key => 'PUBLISH_HDD_1',
+          value => 'openSUSE-%VERSION%-%ARCH%-textmode_with_ltp_networking_installed.qcow2'},
+         {key => 'LTP_SETUP_NETWORKING',    value => 1},
+         {key => 'VIRTIO_CONSOLE',          value => 1},
```

Comment:
I have been adding the `VIRTIO_CONSOLE` var to the machine type, instead of the test suite. The test should be able to work with or without it.